infra/capi/nephio-workload-cluster/pv-crds.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: PackageVariant
+metadata:
+  name: example-crds
+spec:
+  annotations:
+    approval.nephio.org/policy: initial
+  upstream:
+    package: nephio-workload-crds
```

Comment:
@electrocucaracha This PV won't work in this PR because I need to create a separate YAML to register this repository. We can merge this PR and I will do that in another PR in `stock-repo`

Reply:
Ideally, commits should be atomic, in other words, they shouldn't break anything. What about adding that repo in this PR?

Reply:
Yeah sure, I will add it here itself.

Reply:
@electrocucaracha Now I guess this PR is sort of complete.